flink-runtime/src/test/java/org/apache/flink/runtime/rpc/RpcCompletenessTest.java

```diff
@@ -337,4 +337,25 @@ private static boolean isRpcTimeout(Annotation[] annotations) {
 			throw new RuntimeException("Could not retrive basic type information for primitive type " + primitveType + '.');
 		}
 	}
+
+	/**
+	 * Extract all rpc methods defined by the gateway interface
+	 * @param interfaceClass the given rpc gateway interface
+	 * @return all methods defined by the given interface
+	 */
+	private List<Method> getRpcMethodsFromGateway(Class<? extends RpcGateway> interfaceClass) {
```

Comment:
Here we would simply test if `interfaceClass` is of type `RpcGateway` and if so, then return an empty list.